### ISSUE TITLE
Problem deploying restructure to Azure

### ISSUE THREAD

**Issue by justSteve** [enkodellc/blazorboilerplate#204]
I'm repeating @soomon's 'deploy to azure' walkthru - this time starting with the **restructure** branch but am hitting:

```
Could not find 'aspnetcorev2_inprocess.dll'. Exception message:
Error:
  An assembly specified in the application dependencies manifest (BlazorBoilerplate.Server.deps.json) was not found:
    package: 'Microsoft.AspNetCore.Components.WebAssembly', version: '3.2.0-preview2.20160.5'
    path: 'lib/netstandard2.1/Microsoft.AspNetCore.Components.WebAssembly.dll'

Process Id: 4056.
File Version: 13.0.19218.0. Description: IIS ASP.NET Core Module V2. Commit: 4a42afc5aea63750638e118560d43db04bd9ccc2
```

That version is present in .Server's packages folder - I'm stumped on what else to check.

**Comment by soomon** [comment]
I can confirm the issue. Maybe azure is not yet compatible with the new previews.

**Comment by enkodellc** [comment]
@justSteve Might want to search other forums and or try to deploy a basic Blazor app to determine if it is a BB issue or an Azure issue.

**Comment by justSteve** [comment]
@justSteve Likely is not a BB issue. The following from Blazor's gitter:

justSteve @justSteve 07:03
Up to this point it's been said that choosing the 3.0 runtime stack was good enough (and was the only way to get a Windows OS). Perhaps I need to try the 3.1 LTS?

Simon Ziegler @simonziegler 07:05
@justSteve, I found that Azure tends not to have the latest builds available. I solved this by building a docker container with my app and deploying that to Azure.
But your point about wanting Windows OS does of course mean you cannot use docker

justSteve @justSteve 07:07
@simonziegler yeah -- windows = preferred debugging platform.

Jan-Willem Spuij @jspuij 07:10
@justSteve Yes you probably need 3.1 LTS

justSteve @justSteve 07:14
thx - will give it a try.
